### PR TITLE
Bluetooth: tester: enable VCS volume flags notify

### PR DESCRIPTION
**Pull request by ChangNice**
[Description]
PTS case: VCS/SR/SGGIT/CHA/BV-03-C fail.

[Root Cause]
VCS/SR/SGGIT/CHA/BV-03-C will check Volume Flags Characteristic properties (Read/Notify), but tester volume flags notify is disabled by default.

[Fix]
add CONFIG_BT_VCP_VOL_REND_VOL_FLAGS_NOTIFIABLE=y in tests/bluetooth/tester/overlay-le-audio.conf

[Testing]
After modified, VCS/SR/SGGIT/CHA/BV-03-C pass with log:
	189	0	Test Case Ended	VCS/SR/SGGIT/CHA/BV-03-C	PASS	24	 00:00:01.010000	12/9/2024 11:31:55.027000 AM

### CODE REVIEW

**Review by Thalley — APPROVED** [review]

**Review by sjanc — APPROVED** [review]

**Comment by ChangNice** [comment]
Can anyone help merge this PR? I don't have merge permissions.
@Thalley @sjanc

**Comment by Thalley** [comment]
> Can anyone help merge this PR? I don't have merge permissions. @Thalley @sjanc

It will get merged once it has passed the required time in review: https://docs.zephyrproject.org/latest/project/dev_env_and_tools.html#give-reviewers-time-to-review-before-code-merge

**Pull request merged**
